libs/navigation/website/src/lib/footer/footer.tsx

```diff
@@ -1,66 +1,210 @@
+import {FullNavigationFragment} from '@wepublish/website/api'
 import {BuilderFooterProps, useWebsiteBuilder} from '@wepublish/website/builder'
 import {navigationLinkToUrl} from '../link-to-url'
 
 export const FooterWrapper = styled('footer')`
-  align-self: flex-end;
-  color: ${({theme}) => theme.palette.secondary.contrastText};
-  background-color: ${({theme}) => theme.palette.secondary.main};
-  padding: ${({theme}) => theme.spacing(2)};
+  position: sticky;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 10;
 `
 
-export const FooterInnerWrapper = styled(Container)`
+export const FooterInnerWrapper = styled(Toolbar)`
   display: grid;
-  gap: ${({theme}) => theme.spacing(4)};
-  justify-content: center;
   align-items: center;
+  grid-auto-flow: column;
+  justify-content: space-between;
+  justify-items: center;
 
   ${({theme}) => css`
     ${theme.breakpoints.up('md')} {
-      justify-content: space-between;
-      grid-template-columns: auto auto;
+      grid-auto-columns: 1fr;
     }
   `}
 `
 
-export const FooterLinks = styled('nav')`
+export const FooterMain = styled('div')`
+  display: flex;
+  flex-flow: row wrap;
+  align-items: center;
+  justify-self: start;
+  gap: ${({theme}) => theme.spacing(2)};
+`
+
+export const FooterMainItems = styled('div')<{show: boolean}>`
+  display: none;
+  grid-auto-flow: column;
+  grid-auto-columns: max-content;
+  gap: ${({theme}) => theme.spacing(2)};
+  font-weight: ${({theme}) => theme.typography.fontWeightMedium};
+  font-size: 1.125rem;
+  text-transform: uppercase;
+
+  ${({theme, show}) => css`
+    ${theme.breakpoints.up('sm')} {
+      display: ${show ? 'none' : 'grid'};
+    }
+  `}
+`
+
+export function Footer({className, categorySlugs, slug, data, loading, error}: BuilderFooterProps) {
+  const mainItems = data?.navigations?.find(({key}) => key === slug)
+
+  const categories = categorySlugs.map(categorySlugArray => {
+    return categorySlugArray.reduce((navigations, categorySlug) => {
+      const navItem = data?.navigations?.find(({key}) => key === categorySlug)
+
+      if (navItem) {
+        navigations.push(navItem)
+      }
+
+      return navigations
+    }, [] as FullNavigationFragment[])
+  })
+
+  return (
+    <FooterWrapper className={className}>
+      <FooterPaper main={mainItems} categories={categories} />
+    </FooterWrapper>
+  )
+}
+
+const FooterPaperWrapper = styled('div')`
+  padding: ${({theme}) => theme.spacing(2.5)};
+  background-color: ${({theme}) => theme.palette.grey[800]};
+  color: ${({theme}) => theme.palette.primary.contrastText};
   display: grid;
-  grid-template-columns: 1fr 1fr;
-  column-gap: ${({theme}) => theme.spacing(4)};
-  row-gap: ${({theme}) => theme.spacing(2)};
+  gap: ${({theme}) => theme.spacing(3)};
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  transform: translateY(100%);
+  overflow: hidden;
+  grid-row-gap: ${({theme}) => theme.spacing(8)};
```

Comment:
`grid-row-gap` & `grid-column-gap` has been deprecated. It's just `row-gap` now.